.github/container/manifest.yaml

```diff
@@ -40,6 +40,7 @@ paxml:
   patches:
     pull/46/head: file://patches/paxml/PR-46.patch # adds Transformer Engine support
     pull/93/head: file://patches/paxml/PR-93.patch # exposes TE DPA argument in Grok to make it configurable from the command line
+    pull/98/head: file://patches/paxml/PR-98.patch # Add cuda profiler hook function call to paxml 
```

Comment:
You forgot to add the new file.

Reply:
You mean the patch file(patches/paxml/PR-98.patch)? I understand that this file is augo-generated, do we need to check-in it?

Reply:
It's okay, the one we check in is a relic of the old behavior. It's optional now b/c the CI always bumps and adds a new one into it.

BTW, if you ever need to copy a patch file for posterity, you can refer to a nightly or one of the auto generated nightly PRs (example: https://github.com/NVIDIA/JAX-Toolbox/pull/1094)